src/native/minipal/random.h

```diff
@@ -3,14 +3,15 @@
 
 #pragma once
 
-#include <libs/Common/pal_compiler.h>
+#include "compiler.h"
```

Comment:
This shouldn't need a copy of `compiler.h`. I'm assuming this was done for `PALEXPORT` - which should go away if we leave the export / entry point in libs/System.Native.